accounts-db/src/accounts_db.rs

```diff
@@ -4347,6 +4347,16 @@ impl AccountsDb {
                 slot, shrink_collect.alive_total_bytes, shrink_collect.capacity
             );
 
+            if !shrink_collect.all_are_zero_lamports {
```

Comment:
ah, answer is to remove the `info` above on line 4345. This is the one that goes in the `if`. Sorry - I misread the original pr.

Reply:
updated.